app/models/queue_item.rb

```diff
@@ -0,0 +1,22 @@
+class QueueItem < ActiveRecord::Base
+  belongs_to :video
+  belongs_to :user
+
+  def video_title
+    self.video.name
+  end
+
+  def rating
+    review = Review.where(user_id: user.id, video_id: video.id).first
+    review.rating if review
+  end
+
+  def category_name
+    self.video.category.name
```

Comment:
No need to use `self` here. You are using `#video` getter provided by `belongs_to :video`. If you would set the video `#video=` then you would have to use `self`.